docs/dApps/unity/quickstart.md

```diff
@@ -0,0 +1,373 @@
+
+1. Switch the wallet to Ghostnet and use the [Ghostnet faucet](https://faucet.ghostnet.teztnets.xyz/) to get some tez that you can use to send transactions to your Unity projects.
+
+1. Install the SDK from GitHub:
+
+   1. From the SDK's GitHub page at https://github.com/trilitech/tezos-unity-sdk/releases, download the latest release of the SDK, which is a file named `TezosUnitySdk.unitypackage`.
+   1. In the Unity Editor, click **Assets > Import Package > Custom Package** and import the SDK file.
+   1. Verify that the SDK is installed by opening the Project panel and expanding the **Assets > TezosSDK** folder.
+
+     If you see the TezosSDK folder with sub-folders including Editor, Examples, Resources, and Runtime, the SDK is installed correctly, as in this picture:
+
+     <img src="/img/dApps/unity-installed-project-panel.png" alt="The SDK assets in the Project panel" style={{width: 200}} />
+
+1. Verify that the SDK works in your project by running the WalletConnection example scene, which demonstrates how to connect to a Tezos wallet in a Unity project:
+
+   1. In the Project panel, expand **Assets > TezosSDK > Examples > WalletConnection** and open the `_WalletConnection` scene.
+   1. Run the scene, which shows a QR code.
+   1. Use your wallet app to scan the barcode and confirm the connection to the Unity project.
+   The project UI shows the address of the connected account, as in this picture:
+
+      <img src="/img/dApps/unity-wallet-connection-scene-connected.png" alt="The new RPC node selected in the Temple wallet" style={{width: 500}} />
+
+## Connecting to wallets
+
+Connecting to a user's wallet is a prerequisite to working with Tezos in any application.
+Accessing the wallet allows your project to see the tokens in it and to prompt the user to submit transactions, but it does not give your project direct control over the wallet.
+Users must still confirm all transactions in their wallet application.
+
+Using a wallet application in this way saves you from having to implement payment processing and security in your application.
+Game developers can also use the wallet and its account as a unique account identifier and as the user's inventory.
+
+1. Copy the `MainThreadExecutor`, `TezosAuthenticator`, and `TezosManager` prefabs to your scene.
+These prefabs provide prerequisites to use Tezos in a scene.
+
+   The `TezosAuthenticator` prefab automatically adds features that connect to users' wallets.
+   If you copy these prefabs into your scene and run it, it shows a QR code that Tezos wallet applications can scan to connect with the application.
+   You can access these features through the prefab and change how the project manages its connection to users' wallets.
+
+   For example, the `QRCode` object receives a handshake from the `TezosManager` object and uses the handshake information to generate a URI and encode that URI to a QR code image:
```

Comment:
I think we should skip QR code information as it's a bit confusing for newcomers.